charon/config.py

```diff
@@ -24,6 +25,7 @@
 CONFIG_FILE = "charon.yaml"
 
 logger = logging.getLogger(__name__)
+add_file_handler(logger)
```

Comment:
Maybe we should extract these two lines together in logs.py as a new method like get_charon_logger(name: str)? So if we want to change something in future, we don't need to change it everywhere.

Reply:
@ligangty done, wrap it in the getLogger